web/themes/ventuno/src/scss/components/_navbar.scss

```diff
@@ -0,0 +1,315 @@
+.joinup-navbar {
+  background-color: $primary;
+  box-shadow: $box-shadow;
+  padding: 0;
+
+  @include media-breakpoint-between(xs, md) {
+    // height: 54px;
+  }
+
+  @include media-breakpoint-up(md) {
+    height: 72px;
+  }
+
+  .navbar-brand {
+    font-size: .75rem; // 12px
+  }
+
+  .navbar-brand-image {
+
+    @include media-breakpoint-between(xs, md) {
+      width: 6rem; // 96px
+    }
+
+    @include media-breakpoint-up(md) {
+      width: 8rem; // 128px
+    }
+  }
+
+  .navbar-motto {
+
+    @include media-breakpoint-between(xs, md) {
+      display: none;
+    }
+
+    @include media-breakpoint-up(md) {
+      font-size: .875rem; // 14px
+      color: $white;
+      letter-spacing: .1rem; // 1.6px
+    }
+
+  }
+
+  .navbar-toggler {
+    padding: $spacer*.75 0; // 12px 0
+    color: $white;
+    border: none;
+    border-radius: 0;
+
+    &:focus {
+      outline: none;
+      box-shadow: 0 -4px 0 0 $white inset;
+    }
+
+    &:not([aria-expanded="false"]) {
+      box-shadow: 0 -2px 0 0 $white inset;
+    }
+
+    .navbar-toggler-icon {
+      background-image: url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 30 30'%3E%3Cpath stroke='rgba(255, 255, 255, 1)' stroke-linecap='round' stroke-miterlimit='10' stroke-width='2' d='M4 7h22M4 15h22M4 23h22'/%3E%3C/svg%3E");
+    }
+
+  }
+
+  .navbar-collapse {
+    border-bottom-right-radius: $border-radius;
+    border-bottom-left-radius: $border-radius;
+  }
+
+  .navbar-nav {
+    justify-content: flex-end;
+    width: 100%;
+  }
+
+  .nav-item {
+    list-style: none;
+
+    @include media-breakpoint-up(md) {
+      padding: 0 $spacer*.5; // 0 8px
+      &:last-of-type {
+        padding-right: 0;
+      }
+    }
+  }
+
+  .nav-link {
+    font-weight: $font-weight-medium;
+    color: $white;
+
+    @include media-breakpoint-between(xs, md) {
+      padding: $spacer*.75 0; // 12px 0
+    }
+
+    @include media-breakpoint-up(md) {
+      padding: $spacer*1.5 0; // 24px 8px
+    }
+
+    &:hover {
+      color: $gray-200;
+    }
+
+  }
+
+  .accordion-collapse {
+    position: absolute;
+    z-index: 1;
+    background: $primary;
+    border-bottom-right-radius: $border-radius;
+    border-bottom-left-radius: $border-radius;
+    box-shadow: $box-shadow;
+    margin-top: 1px;
+
+    @include media-breakpoint-between(xs, md) {
+      padding: $spacer*.875 $spacer; // 14px 16px
+    }
+
+    @include media-breakpoint-up(md) {
+      max-width: 20*$spacer; // 320px
+      padding: $spacer $spacer*1.5; // 16px 24px
+    }
+
+    @include media-breakpoint-between(xs, sm) {
+      right: $spacer*.75;
+    }
+
+    @include media-breakpoint-between(sm, md) {
+      right: calc(50% - 258px);
```

Comment:
Not sure if we can get rid of these fixed values, but it might be a problem with real world contents, translations etc.